config/vendor-api-v1.yml

```diff
@@ -394,6 +394,7 @@ components:
           - pending_conditions
           - recruited
           - rejected
+          - rejected_at_end_of_cycle
```

Comment:
I don't think we can add this here at this stage unfortunately, as it could be a breaking change. I think we ought to map to `rejected` for now and consider adding a boolean `rejected_at_end_of_cycle` field if this information turned out to be useful to vendors. Adding a card for this status in our draft API v2 feature list https://trello.com/c/4PcIvMCC/704-add-rejectedatendofcycle-status

Reply:
Fixt!